### ISSUE TITLE
Can local elevation data be loaded offline

### ISSUE THREAD

**Issue by haloihaloi123** [SimBlocks/OneWorldSDKforUnity#20]

**Comment by SimBlocks** [comment]
Yes, this feature is available in the Professional version of the One World SDK for Unity, which is available from SimBlocks.

**Comment by haloihaloi123** [comment]
Can THIS version load WMTS data

**Comment by haloihaloi123** [comment]
> Yes, this feature is available in the Professional version of the One World SDK for Unity, which is available from SimBlocks.

Can THIS version load WMTS data

**Comment by SimBlocks** [comment]
WMTS is currently supported for imagery.
